editor/views/record_sets.py

```diff
@@ -324,8 +334,14 @@ def _render_left_panel():
             )
             data_editor_key = _data_editor_key(record_set_key, record_set)
             st.markdown(
-                f"{needed_field('Fields')} (add/delete fields by directly editing the"
-                " table)"
+                needed_field("Fields"),
+                help=(
+                    "Add/delete fields by directly editing the table. Warning: the"
+                    " table contains information about the fields--not the data"
+                    " directly. If you wish to embed data, select `The RecordSet is an"
+                    " enumeration` above. To edit fields details, click the button"
```

Comment:
select `The RecordSet is an  enumeration` above

this sounds a bit strange to me (maybe `from above` ? or `tick the `this ...is an enumeration` box?) but as a non-native speaker I'm also not 100% sure

Reply:
Done.